packages/editor/src/lib/config/TLUserPreferences.ts

```diff
@@ -139,7 +139,7 @@ function storeUserPreferences() {
 		window.localStorage.setItem(
 			USER_DATA_KEY,
 			JSON.stringify({
-				version: userTypeMigrations.currentVersion,
+				version: userMigrations.currentVersion,
```

Comment:
I dunno how this got started, but migrations and validations had `Type` in their names. Some validators actually were for a specific type, e.g. the type of a geo shape, but the rest were unclear whether they were referring to `RecordType`s or something else. Either way, all those `xxxxTypeMigrations` are here changed to `xxxxMigrations`.